tutorial/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.21)
+
+project(
+  musica-demo
+  VERSION 0.1
+  LANGUAGES CXX C Fortran
+)
+
+# mkdir build
+# cd build
+# cmake -DMUSICA_INSTALL_DIR=<musica_install_dir> ..
```

Comment:
Doesn't look like there is `MUSICA_INSTALL_DIR` option. Is it still valid? I also think it could be omitted since `chapter0.rst` covers the build process. What do you think?

Reply:
MUSICA_INSTALL_DIR is not defined during the MUSICA configuration, but rather during the host model configuration via the cmake -D option, and points to the installation location of the MUSICA library.

Reply:
Ah, I see. I didn't know that